benchmarks/htap/htap_loader.py

```diff
@@ -0,0 +1,306 @@
+from datetime import datetime
+from io import StringIO, SEEK_SET
+
+from psycopg2.extras import execute_values
+
+from s64da_benchmark_toolkit.dbconn import DBConn
+
+from benchmarks.htap.lib.helpers import (
+        Random, TPCCText, TPCHText, NATIONS, REGIONS, TimestampGenerator
+)
+
+MAXITEMS = 100000
+DIST_PER_WARE = 10
+CUST_PER_DIST = 3000  # Standard says: 30k customers total (for all districts)
+NUM_ORDERS = 3000
+STOCKS = 100000
+NUM_SUPPLIERS = 10000
+NUM_NATIONS = 62
+NUM_REGIONS = 5
+
+
+class TPCCLoader():
+    def __init__(self, dsn, w_id, scale_factor, start_date=None):
+        self.dsn = dsn
+        self.w_id = w_id
+        self.random = Random(seed=w_id)
+        self.tpcc_text = TPCCText(self.random)
+        self.tpch_text = TPCHText(self.random)
+        self.start_date = start_date or datetime.now()
+        self.timestamp_generator = TimestampGenerator(self.start_date, scale_factor, seed=w_id)
```

Comment:
Is there a reason we pass here a seed instead of a `Random` object like in `TPCCText` and TPCHText`? If there's no real reason, it would be nice to unify the way we deal with random in these classes.

Reply:
done